client/src/components/TimeZoneDropdown.js

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react';
+import { Dropdown } from 'semantic-ui-react'
+import { makeCall } from '../apis'
+
+var timeZoneOptions = [
```

Comment:
Here are the tz options - I think it needs reworking but I'm not sure how to approach it

Reply:
What reworkings did you have in mind?